keyboards/sharkoon/skiller_sgk50_s4/info.json

```diff
@@ -0,0 +1,225 @@
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [2, 1], "x": 17, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 34, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 52, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 69, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 86, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 103, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 121, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 138, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 155, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 172, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 190, "y": 32, "flags": 4},
+            {"matrix": [2, 13], "x": 207, "y": 32, "flags": 4},
+            {"matrix": [2, 12], "x": 224, "y": 16, "flags": 4},
+            {"matrix": [1, 12], "x": 207, "y": 16, "flags": 4},
+            {"matrix": [1, 11], "x": 190, "y": 16, "flags": 4},
+            {"matrix": [1, 10], "x": 172, "y": 16, "flags": 4},
+            {"matrix": [1, 9], "x": 155, "y": 16, "flags": 4},
+            {"matrix": [1, 8], "x": 138, "y": 16, "flags": 4},
+            {"matrix": [1, 7], "x": 121, "y": 16, "flags": 4},
+            {"matrix": [1, 6], "x": 103, "y": 16, "flags": 4},
+            {"matrix": [1, 5], "x": 86, "y": 16, "flags": 4},
+            {"matrix": [1, 4], "x": 69, "y": 16, "flags": 4},
+            {"matrix": [1, 3], "x": 52, "y": 16, "flags": 4},
+            {"matrix": [1, 2], "x": 34, "y": 16, "flags": 4},
+            {"matrix": [1, 1], "x": 17, "y": 16, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 16, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 4},
+            {"matrix": [0, 1], "x": 17, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 34, "y": 0, "flags": 4},
+            {"matrix": [0, 3], "x": 52, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 69, "y": 0, "flags": 4},
+            {"matrix": [0, 5], "x": 86, "y": 0, "flags": 4},
+            {"matrix": [0, 6], "x": 103, "y": 0, "flags": 4},
+            {"matrix": [0, 7], "x": 121, "y": 0, "flags": 4},
+            {"matrix": [0, 8], "x": 138, "y": 0, "flags": 4},
+            {"matrix": [0, 9], "x": 155, "y": 0, "flags": 4},
+            {"matrix": [0, 10], "x": 172, "y": 0, "flags": 4},
+            {"matrix": [0, 11], "x": 190, "y": 0, "flags": 4},
+            {"matrix": [0, 12], "x": 207, "y": 0, "flags": 4},
+            {"matrix": [0, 13], "x": 224, "y": 0, "flags": 4}
+        ],
+        "max_brightness": 110,
+        "sleep": true,
+        "val_steps": 28
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1020",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
```suggestion
        "LAYOUT": {
```

Reply:
@waffle87 this has multiple layouts, so doesn't the "all" apply here?